index.html

```diff
@@ -40,6 +40,86 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+    <!-- Work -->
+    <section id="work">
+      <div class="work-first">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <P>&#160;&#160;&#9899;&#160;&#160;Full Stack Dev</P>
+                <p>&#160;&#160;&#9899;&#160;&#160;2023</p>
+              </div>
+             
+            </div>
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description">
+          <button class="language-btn">html</button>
+          <button class="language-btn">css</button>
+          <button class="language-btn">Javascript</button>
+        </div>
```

Comment:
- [ ] Please use `ul` and `li` for lists because it is semantically correct and good for search engine optimization so kindly use `ul` and 'li' instead of `div` and `buttons`

```suggestion
        <ul class="work-description">
          <li class="language-btn">html</button>
          <li class="language-btn">css</button>
          <li class="language-btn">Javascript</button>
        </ul>
```

- [ ] And style same as Figma design :slightly_smiling_face: